### PR TITLE
Changed stylingintraslation

### PR DESCRIPTION
**Pull request by aditya-369**

### CODE REVIEW

**Comment by aditya-369** [comment]
Learning git so please excuse.I will clean all these pull reqs

**Comment by shardulc** [comment]
@aditya-369 No problem, git can indeed be a little confusing at first in such cases. I recommend creating a new branch *from master* (not from one of your existing branches), adding your changed files there, and creating a pull request.

File: assets/css/translation.css [hunk 1]
```diff
@@ -126,3 +126,159 @@ html[dir='rtl'] #srcLanguages {
...
+
+#footer-container-unoderlist {
+    display: inline-block;
+}
+
+#mistakeId {
+    cursor: pointer;
+}
+
+#divForMistake {
```
Comment (sushain97) [inline assets/css/translation.css]: There's no need for any conjunctions inside ids. `mistakeDiv`. In fact, I don't think there should be any element names in ids either. `mistakeContainer`.

Comment (aditya-369) [reply assets/css/translation.css]: Sorry @sushain97 did not rectify mistakes as said before.Will look over it.Thanks

**Comment by aditya-369** [comment]
#304

**Pull request closed without merging**